Sources/App/Services/Helper/QuestionnaireService.swift

```diff
@@ -0,0 +1,55 @@
+//
+// This source file is part of the ENGAGE-HF-AI-Voice open source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import ModelsR4
+import Vapor
+
+
+/// Protocol for questionnaire services.
+/// It delegates the management of the questionnaire response to the `QuestionnaireManager` and the persistent storage of the questionnaire response to the `QuestionnaireStorageService`.
+@MainActor
+protocol QuestionnaireService {
```

Comment:
I think we should ensure that we move away from a static setup to a setup where a service is created and distorted with every web socket creation to ensure that we can support parallel requests. The questionnaire manager should be injected using, e.g., a dependency injection mechanism or even simpler for now just passed into the initializer of the service.

Reply:
Creating the questionnaire manager and storage service upon initialization of the service now.